src/test/e2e-emulated/contracts/base-trait-constant-override-1.tact

```diff
@@ -0,0 +1,13 @@
+trait OverrideBaseTraitConstant {
+    override const storageReserve: Int = 100;
+}
```

Comment:
Should not allow implicit inheritance of `BaseTrait` by other traits. Only contracts should inherit from `BaseTrait`. So `override` here should be an error

Reply:
Done, now we need an explicit `with BaseTrait` for traits to use `BaseTrait` functions and constants